Microsoft.Toolkit.Uwp.Services/Services/Bing/BingSearchConfig.cs

```diff
@@ -28,5 +28,10 @@ public class BingSearchConfig
         /// Gets or sets search query.
         /// </summary>
         public string Query { get; set; }
+
+        /// <summary>
+        /// Gets or sets search query news.
+        /// </summary>
+        public bool News { get; set; }
```

Comment:
On second thought... perhaps an enum would be better here? That also allows us to extend if Bing search extends.